### ISSUE TITLE
Vue weights 120KB minified in production build

### ISSUE THREAD

**Issue by PierBover** [vuejs-templates/webpack#782]
So I started analysing why my project was much more heavy than expected, and after using [webpack-visualizer](https://github.com/chrisbateman/webpack-visualizer) Vue seems to weight about 120KB minified after `npm run build`.

![image](https://user-images.githubusercontent.com/226483/27765065-2257b4bc-5e6e-11e7-9436-8396a582fd1d.png)

Is this normal?

Seems a lot compared to what's claimed on the Vue website.

> 19kb min+gzip Runtime

**Comment by MaxMilton** [comment]
Have you made any changes to the template?

With all the default options I get (build code; minified, gzip'd):
Runtime + compiler: 28.2 KB
Runtime only: 19.5 KB

The template already comes with `webpack-bundle-analyzer` which can be used via `npm run build --report`.

**Comment by PierBover** [comment]
I created a new project from scratch using the latest version and and I get your numbers.

Then I tried `npm run build --report` on my project and the numbers differ from Webpack Visualizer.

I get these numbers which are closer to what I was expecting, but still higher than what I'm getting when building from a new template.

![image](https://user-images.githubusercontent.com/226483/27776634-0837547a-5f72-11e7-8f00-32ad778879c7.png)

I guess I should migrate the project to a new version of the template.
